app/components/search-facet-provider.js

```diff
@@ -30,82 +31,78 @@ var getProvidersPayload = '{"from": 0,"query": {"bool": {"must": {"query_string"
 export default Component.extend(Analytics, {
     store: service(),
     theme: service(),
-    otherProviders: [],
-    searchUrl: config.OSF.shareSearchUrl,
     whiteListedProviders: config.whiteListedProviders.map(item => item.toLowerCase()),
-    osfUrl: config.OSF.url,
-
     init() {
         this._super(...arguments);
+        this.set('otherProviders', []);
         Promise.all([
             // The providers list from the API
             this.get('store')
                 .findAll('preprint-provider')
-                .then(providers => {
-                    const providerNames = providers.filter(
-                        provider => provider.get('id') !== 'livedata'
-                    ).map(provider => {
-                        const name = provider.get('shareSource') || provider.get('name');
-                        // TODO Change this in populate_preprint_providers script to just OSF
-                        return name === 'Open Science Framework' ? 'OSF' : name;
-                    });
-                    this.set('osfProviders', providerNames);
-                    return providerNames;
-                }),
+                .then(this._filterProviders.bind(this)),
             // The providers list from SHARE
+            /* eslint-disable-next-line ember/named-functions-in-promises */
```

Comment:
not sure why this was was disabled

Reply:
Possibly a lost disable? It could have been for a few lines down.